src/raftstore/store/fsm/peer.rs

```diff
@@ -1601,10 +1610,19 @@ impl<T: Transport, C: PdClient> Store<T, C> {
 
         let mut total_gc_logs = 0;
 
+        thread_local! {
```

Comment:
don't use the thread local here. 

I don't think that sending in every compact tick will cause the performance problem.

Reply:
I just want to reduce the times of memory allocations while putting element into `gc_vec`. In stress test, the numbers of entry cache needed to be reclaimed are similar. So, I choose to use the length of last `gc_vec` and store this info by `thread_local`.

Reply:
Isn't it better to alloc a little larger than that allocated last time? The amount of entry caches to gc won't be perfectly stable.
For example, if we run it for 5 times and we got the vec's len to be `100, 101, 99, 100, 101` respectively, it will cause reallocation at the 2nd, 4th, 5th time, because it's larger than the previous one. However if you reserve `last_time_size + 10`, then these reallocations can be avoided.

Reply:
I guess maybe `last_time_size * 1.1` is better in your solution.

Reply:
I have extracted these info in `Store` instead of `thread_local`.